packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseParameterObject-test.js

```diff
@@ -144,6 +144,56 @@ describe('Parameter Object', () => {
     });
   });
 
+  describe('#example', () => {
+    it('attaches string example to member', () => {
+      const parameter = new namespace.elements.Object({
+        name: 'example',
+        in: 'query',
+        example: 'example_value',
+      });
+
+      const parseResult = parse(context, parameter);
+
+      expect(parseResult.length).to.equal(1);
+      expect(parseResult.get(0)).to.be.instanceof(namespace.elements.Member);
+      expect(parseResult.get(0).value.toValue()).to.equal(
+        'example_value'
```

Comment:
Why the extra lines here?

Reply:
@pksunkara should I rewrite it to one line?